tropescraper/use_cases/parsers/tvtropes_parser.py

```diff
@@ -29,7 +29,8 @@ class TVTropesParser(object):
                               'SportingEvent', 'RecordedAndStandUpComedy', 'AnimatedShows', 'BroadcastLive',
                               'TelevisionMovieIndex', 'Series', 'Theatre', 'Ballet', 'Opera', 'Vaudeville',
                               'ThemeParks', 'Toys', 'FairyTale', 'FranchiseIndex', 'Legend', 'Mythology',
-                              'Analysis', 'UsefulNotes'}
+                              'Analysis', 'UsefulNotes', 'Recap', 'WMG', 'VideoExamples','FanficRecs',
```

Comment:
👍 Perfect! The new additions to the media types in the black list make a lot of sense.